vllm_ascend/core/scheduler.py

```diff
@@ -15,20 +15,18 @@
 # This file is a part of the vllm-ascend project.
```

Comment:
please re-enable AscendScheduler test 
https://github.com/vllm-project/vllm-ascend/blob/main/.github/workflows/vllm_ascend_test.yaml#L111-L112  @zzzzwwjj  please take a look as well.  There is a pr for scheduler as well https://github.com/vllm-project/vllm-ascend/pull/939